test/lexer/Lexer.test.js

```diff
@@ -341,4 +341,38 @@ describe("lexer", () => {
             ]);
         });
     });
+
+    describe('two word keywords', () => {
+        it('supports various spacing between for each', () => {
+            var k = 2;
```

Comment:
Doesn't look like this (or the other `var k = 2` on line 362) are being used.  Would you mind removing them?

Reply:
oops. Yeah, my bad. The typescript debugger doesn't always break on the line when you have `let {tokens}`, so I had to break the line before.

Reply:
No worries, friend!

Reply:
Removed!